src/ops/controlflow.rs

```diff
@@ -190,43 +180,73 @@ impl OpTrait for BasicBlock {
     }
 
     fn dataflow_signature(&self) -> Option<FunctionType> {
-        Some(match self {
-            BasicBlock::DFB {
-                extension_delta, ..
-            } => FunctionType::new(type_row![], type_row![]).with_extension_delta(extension_delta),
-            BasicBlock::Exit { .. } => FunctionType::new(type_row![], type_row![]),
-        })
+        Some(
```

Comment:
Hmmm, I highly suspect this should be `None` (but instead we should implement `fn extension_delta` which by default delegates to this). However, that's not really part of this PR